### ISSUE TITLE
docs problem

### ISSUE THREAD

**Issue by ChenXiaojun2008** [ultralytics/ultralytics#4164]
link: https://docs.ultralytics.com/modes/predict/#probs
![image](https://github.com/ultralytics/ultralytics/assets/76811041/5880b47f-e815-4c08-b2e2-fbb574f8e2ad)
I found some issues in this sample program.
```
from PIL import Image
from ultralytics import YOLO

# Load a pretrained YOLOv8n model
model = YOLO('yolov8n.pt')

# Run inference on 'bus.jpg'
results = model('bus.jpg')  # results list

# Show the results
for r in results:
    im_array = r.plot()  # plot a BGR numpy array of predictions

    # im = Image.fromarray(im[..., ::-1])  # RGB PIL image
    im = Image.fromarray(im_array[..., ::-1])  # RGB PIL image

    im.show()  # show image
    im.save('results.jpg')  # save image
```

**Comment by glenn-jocher** [comment]
@ChenXiaojun2008 thank you for bringing this to our attention!

In the code snippet provided, I see that you are using the `YOLO()` function to load a pre-trained YOLOv8n model and then running inference on an image named "bus.jpg". Afterward, you are iterating through the results to plot and save the image.

If you're running into issues, it might be due to one of the following reasons:

1. **Model File**: Make sure that the 'yolov8n.pt' model file is in the correct directory and the path to the file is correctly specified. If your model file is located somewhere else, you will need to provide the full path.

2. **Input Image**: The image 'bus.jpg' should also be in the correct directory, or a full path should be specified if it's located in a different directory.

3. **Result Object**: The `model()` function should return a Result object that contains inference results, which include bounding boxes, labels, and confidences, among other things. If you're seeing an error here, there might be an issue with the inference process.

4. **Image Display**: Ensure that you have the necessary permissions to save and display images on your local system. In some environments, attempting to `.show()` or `.save('results.jpg')` an image might throw an error if permissions are not set appropriately.

If you could provide more information about the issues you're encountering with this code, it would be easier to help you troubleshoot. Please include any error messages or descriptions of unexpected behavior you're seeing.

**Comment by glenn-jocher** [comment]
@ChenXiaojun2008 great news 😃! Your original issue may now be resolved ✅ in PR #4180 and deployed in our new pip update as `ultralytics 8.0.148` with updated Docs. 

To get this update:
- **[Git](https://github.com/ultralytics/ultralytics)** – Run `git pull` from within your `ultralytics/` directory or run `git clone https://github.com/ultralytics/ultralytics` again
- **[Pip](https://pypi.org/project/ultralytics/)** – Update with `pip install -U ultralytics`
- **[Notebooks](https://github.com/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb)** – Check out the updated notebooks  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/ultralytics)** – Run `sudo docker pull ultralytics/ultralytics:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and letting us know. Please confirm if this update fixes the issue for you, and don't hesitate to report any other **issues** you find or **feature requests** you may have. Happy training with YOLOv8 🚀!
